ova/provision.sh

```diff
@@ -1,20 +1,35 @@
 #/bin/bash
-set -exf
+set -x
```

Comment:
`set -e` makes the script fail whenever there is an error, is there a reason to remove this option?

Reply:
Done in 039d5eee2611844622e77f70f8d01bfbf9221dcb